relstorage/storage.py

```diff
@@ -1073,6 +1073,9 @@ def _abort(self):
         if self.blobhelper is not None:
             self.blobhelper.abort()
 
+    def afterCompletion(self):
+        self._rollback_load_connection()
```

Comment:
"It"? I assume "it" is _rollback_load_connection, which is fine. But why is _rollback_load_connection not checking _load_transaction_open? Wouldn't that allow it to avoid a server round trip?